### ISSUE TITLE
helm crach after send stop signal

### ISSUE THREAD

**Issue by wirwolf** [SomeBlackMagic/helm-assistant#2]
On send SIGQUIT: quit
```
PC=0x46a7b8 m=0 sigcode=0
goroutine 1 [running, locked to thread]:
runtime.memmove()
	/usr/local/go/src/runtime/memmove_amd64.s:180 +0x138 fp=0xc00018d5a8 sp=0xc00018d5a0 pc=0x46a7b8
runtime.typedmemmove(0x1bf4ce0, 0xc000283b88, 0xc00018d6d0)
	/usr/local/go/src/runtime/mbarrier.go:171 +0x65 fp=0xc00018d5e0 sp=0xc00018d5a8 pc=0x414045
runtime.mapassign(0x1a0fb80, 0xc0000eab10, 0x1cd53d2?)
	/usr/local/go/src/runtime/map.go:680 +0x486 fp=0xc00018d668 sp=0xc00018d5e0 pc=0x40f5e6
github.com/gogo/protobuf/proto.RegisterType({0x20f9558?, 0x0}, {0x1cd53d2, 0x2d})
	vendor/github.com/gogo/protobuf/proto/properties.go:559 +0x279 fp=0xc00018d700 sp=0xc00018d668 pc=0x71ca39
k8s.io/api/authorization/v1beta1.init.0()
	vendor/k8s.io/api/authorization/v1beta1/generated.pb.go:445 +0xb9 fp=0xc00018d730 sp=0xc00018d700 pc=0xdf55d9
runtime.doInit(0x2e24dc0)
	/usr/local/go/src/runtime/proc.go:6321 +0x126 fp=0xc00018d860 sp=0xc00018d730 pc=0x447446
runtime.doInit(0x2e320c0)
	/usr/local/go/src/runtime/proc.go:6298
 +0x71 fp=0xc00018d990 sp=0xc00018d860 pc=0x447391
runtime.doInit(0x2e2ae60)
	/usr/local/go/src/runtime/proc.go:6298 +0x71 fp=0xc00018dac0 sp=0xc00018d990 pc=0x447391
runtime.doInit(0x2e30c40)
	/usr/local/go/src/runtime/proc.go:6298 +0x71 fp=0xc00018dbf0 sp=0xc00018dac0 pc=0x447391
runtime.doInit(0x2e2e620)
	/usr/local/go/src/runtime/proc.go:6298 +0x71 fp=0xc00018dd20 sp=0xc00018dbf0 pc=0x447391
runtime.doInit(0x2e33e20)
	/usr/local/go/src/runtime/proc.go:6298 +0x71 fp=0xc00018de50 sp=0xc00018dd20 pc=0x447391
runtime.doInit(0x2e19920)
	/usr/local/go/src/runtime/proc.go:6298 +0x71 fp=0xc00018df80 sp=0xc00018de50 pc=0x447391
runtime.main()
	/usr/local/go/src/runtime/proc.go:233 +0x1d3 fp=0xc00018dfe0 sp=0xc00018df80 pc=0x43a073
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc00018dfe8 sp=0xc00018dfe0 pc=0x4697e1
goroutine 2 [force gc (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc0000b4fb0 sp=0xc0000b4f90 pc=0x43a476
runtime.goparkunlock(...)
	/usr/local/go/src/runtime/proc.go:369
runtime.forcegchelper()
	/usr/local/go/src/runtime/proc.go:302 +0xad fp=0xc0000b4fe0 sp=0xc0000b4fb0 pc=0x43a30d
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000b4fe8 sp=0xc0000b4fe0 pc=0x4697e1
created by runtime.init.6
	/usr/local/go/src/runtime/proc.go:290 +0x25
goroutine 3 [GC sweep wait]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc0000b5790 sp=0xc0000b5770 pc=0x43a476
runtime.goparkunlock(...)
	/usr/local/go/src/runtime/proc.go:369
runtime.bgsweep(0x0?)
	/usr/local/go/src/runtime/mgcsweep.go:278 +0x8e fp=0xc0000b57c8 sp=0xc0000b5790 pc=0x424fce
runtime.gcenable.func1()
	/usr/local/go/src/runtime/mgc.go:178 +0x26 fp=0xc0000b57e0 sp=0xc0000b57c8 pc=0x419c86
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000b57e8 sp=0xc0000b57e0 pc=0x4697e1
created by runtime.gcenable
	/usr/local/go/src/runtime/mgc.go:178 +0x6b
goroutine 4 [GC scavenge wait]:
runtime.gopark(0xc0000680e0?
, 0x20e5b18?, 0x1?, 0x0?, 0x0?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc0000b5f70 sp=0xc0000b5f50 pc=0x43a476
runtime.goparkunlock(...)
	/usr/local/go/src/runtime/proc.go:369
runtime.(*scavengerState).park(0x2efdc80)
	/usr/local/go/src/runtime/mgcscavenge.go:389 +0x53 fp=0xc0000b5fa0 sp=0xc0000b5f70 pc=0x423073
runtime.bgscavenge(0x0?)
	/usr/local/go/src/runtime/mgcscavenge.go:617 +0x45 fp=0xc0000b5fc8 sp=0xc0000b5fa0 pc=0x423645
runtime.gcenable.func2()
	/usr/local/go/src/runtime/mgc.go:179 +0x26 fp=0xc0000b5fe0 sp=0xc0000b5fc8 pc=0x419c26
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000b5fe8 sp=0xc0000b5fe0 pc=0x4697e1
created by runtime.gcenable
	/usr/local/go/src/runtime/mgc.go:179 +0xaa
goroutine 5 [finalizer wait]:
runtime.gopark(0x43a7f7?, 0x43a245?, 0x0?, 0x0?, 0xc0000b4770?)
	/usr/local/go/src/runtime/proc.go:363 +0xd6 fp=0xc0000b4628 sp=0xc0000b4608 pc=0x43a476
runtime.goparkunlock(...)
	/usr/local/go/src/runtime/proc.go:369
runtime.runfinq()
	/usr/local/go/src/runtime/mfinal.go:180 +0x10f fp=0xc0000b47e0 sp=0xc0000b4628 pc=0x418d8f
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000b47e8 sp=0xc0000b47e0 pc=0x4697e1
created by runtime.createfing
	/usr/local/go/src/runtime/mfinal.go:157 +0x45
rax    0x2116ba8
rbx    0x10
rcx    0x1bc8340
rdx    0x10
rdi    0xc000283b88
rsi    0xc00018d6d0
rbp    0xc00018d5d0
rsp    0xc00018d5a0
r8     0xc000283b88
r9     0xc000283c08
r10    0x0
r11    0x0
r12    0xc000283b80
r13    0xc000283b80
r14    0xc0000061a0
r15    0x0
rip    0x46a7b8
rflags 0x10246
cs     0x33
fs     0x0
gs     0x0
Helm command failed. Exit code:null
```

**Comment by wirwolf** [comment]
fix by https://github.com/SomeBlackMagic/helm-assistant/commit/acae4abfc49a0b6b8bd772cea23c32fb1eec745a

**Comment by wirwolf** [comment]
Fixed in 0.2.4
